keycloak_client.go

```diff
@@ -200,7 +204,10 @@ func (c *Client) GetTokenInfo(realm string, username string, password string, fo
 	var err error
 	var newInfo *TokenInfo
 	key := realm + username
+	KeycloakTokenInfoLock.Lock()
+	// Required exclusive access
 	info, ok := c.tokens[key]
+	KeycloakTokenInfoLock.Unlock()
```

Comment:
ok this is a solution which we can do as a quick hack to get a release out the door today, but afterwards either you or me is going to come back and fix this for real by making the Keycloak client in identity service a non-pointer value so that each go-routine can have its own Keycloak client (it will mean no shared cache but for things that are in a go routine performance is not our primary metric, simplicity and safety are.)